src/hooks/usePropsGetters/useAddToFavoritesButtonProps.ts

```diff
@@ -14,13 +16,16 @@ export default function useAddToFavoritesButtonProps(
     [favorited]
   );
 
+  const isResultFavorited = (result: Partial<QuizResultData>) =>
+    result?.data?.id && favoriteItems?.includes(result?.data?.id);
+
   const getAddToFavoritesButtonProps: GetAddToFavoritesButtonProps = useCallback(
     (result, price) => ({
       className: `${'cio-result-card-favorites-button'} ${
```

Comment:
Completely unrelated to this PR but I noticed while reviewing this, we only change the border color of the heart icon, would it make sense to fill it? I think the current one is hard to notice